README.md

```diff
@@ -295,9 +295,9 @@ Accessing and modifying the default resource is done through two functions:
 #### Example
 
 ```c++
-rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource(); // Points to `cuda_memory_resource`
+rmm::mr::cuda_memory_resource cuda_mr;
```

Comment:
I guess the concern here is the lifetime of this if created on the stack... (This is why Jake created owning_wrapper)

But really I think this is fine -- we don't need to teach users how to manage lifetimes.

Reply:
I agree that there are lifetime issues (where I'm using this, I just made `cuda_mr` and `pool_mr` static), but that was the case before, too, since `pool_mr` has that same issue.

Also, BTW, in terms of your project board, this is still on v0.17.